xml/System.Net.NetworkInformation/Ping.xml

```diff
@@ -1771,6 +1771,17 @@
       </AssemblyInfo>
       <Docs>
         <summary>Sends an Internet Control Message Protocol (ICMP) echo message to a computer, and receives a corresponding ICMP echo reply message from that computer as an asynchronous operation.</summary>
+        <remarks>
+          <format type="text/markdown"><![CDATA[  
+  
+## Remarks  
+ These methods do not cause your application's main thread to block. If you want to block while waiting for the ICMP echo reply message, use the <xref:System.Net.NetworkInformation.Ping.Send%2A> methods.  
```

Comment:
I find this remark a bit strange.  How is this different from any other XxAsync method?